content/2.0/index.html

```diff
@@ -0,0 +1,117 @@
+  <dd>
+    Jenkins 2.0 starts off with much more modern and richer set of functionalities that are catered
+    to today's development best practices. These functionalities are carefully chosen from the vast
+    pool of available Jenkins plugins by Jenkins developers and tested together for better quality.
+
+  <dt>Improved user interfaces
+  <dd>
+    A number of small but useful improvements are made to the user interfaces that
+    you touch everyday, such as the freestyle job configuration pages.
+
+  <dt>All the Jenkins goodness you've accustomed &amp; loves
+  <dd>
+    Jenkins 2.0 is a drop-in replacement of Jenkins 1.x and fully backward compatible.
+    All the plugins and existing configuration should run as is.
+</dl>
+<p>
+  See Wiki for more details of <a href="wiki.jenkins-ci.org/display/JENKINS/Jenkins+2.0">the 2.0 plan</a>
+</p>
+
+
+
+<!-- TODO: how to parse version number into here? -->
+<h2>Preview Binaries (2.0-alpha-1)</h2>
+<dl id=download>
+  <style>
+    #download {
+      padding-left: 1em;
+    }
+    #download DT {
+      font-size: 1.2em;
+      font-weight: bold;
+      margin: 0.2em 0;
+      padding: 0;
+    }
+    #download DD IMG {
+      vertical-align: middle;
+    }
+  </style>
+
+  <dt>Download multi-platform war file</dt>
+  <dd style="margin-bottom:1em">
+    <div>
+      <a href="http://mirrors.jenkins-ci.org/war-rc/latest/jenkins.war"><strong>jenkins.war</strong></a>
+    </div>
+    <!-- TODO: changelog
+    <div class="release-block-small" style="text-align:right">
+     <a href="/changelog" class=release-block-soft>changelog</a> | <a href="http://mirrors.jenkins-ci.org/war/" class=release-block-soft>past releases</a>
+    </div>
+    -->
+  </dd>
+
+  <dt>Or native package</dt>
+  <dd class="release-block-platforms">
+    <div>
+      <img src="/images/os/win_other.png">
+      <a class="release-block-soft" href="http://mirrors.jenkins-ci.org/windows-rc/jenkins-2.0.zip">Windows</a>
+    </div>
+    <div>
+      <img src="/images/os/ubuntu.png">
+      <a class="release-block-soft" href="http://mirrors.jenkins-ci.org/debian-rc/jenkins_2.0_all.deb">Ubuntu/Debian</a>
+    </div>
+    <div>
+      <img src="/images/os/redhat.png">
+      <a class=release-block-soft href="http://mirrors.jenkins-ci.org/redhat-rc/jenkins-2.0-1.1.noarch.rpm">Red Hat/Fedora/CentOS</a>
+    </div>
+    <div>
+      <img src="/images/os/os_macosx.png">
+      <a class=release-block-soft href="http://mirrors.jenkins-ci.org/osx-rc/jenkins-2.0.pkg">Mac OS X</a>
+    </div>
+    <div>
+      <img src="/images/os/opensuse.png">
+      <a class=release-block-soft href="http://mirrors.jenkins-ci.org/opensuse-rc/jenkins-2.0-1.2.noarch.rpm">openSUSE</a>
+    </div>
+  </dd>
+</dl>
+
+
+<h2>Feedback welcome</h2>
+<p>
+  Tell us what you think about 2.0 via
+  <a href="https://twitter.com/jenkinsci">Twitter</a>,
+  <a href="https://groups.google.com/forum/#!forum/jenkinsci-users">email</a>, and
```

Comment:
Alternatively, could we set up a new list for this @rtyler that allows posting without subscribing?